dtfs-central-api/src/v1/routes/tfm-routes.js

```diff
@@ -255,7 +255,7 @@ tfmRouter.route('/deals')
  *   get:
  *     summary: Get TFM facilities associated with a deal ID
  *     tags: [TFM]
- *     description: Get TFM facilities associated with a deal ID. This currenty only works for GEF facilities
+ *     description: Get TFM facilities associated with a deal ID. This currently only works for GEF facilities
```

Comment:
Is this still true, have you got this working for all facilities now? Maybe stupid question

Reply:
this only returns the deals based on a specific `dealId` - i.e. when you are looking on a deal page.